.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -20,36 +27,106 @@ jobs:
     - uses: ./.github/actions/job-preamble
       with:
         gcp-account: ${{ secrets.GCP_SERVICE_ACCOUNT_STACKROX_CI }}
+    - uses: ./.github/actions/cache-go-dependencies
+
+    - name: Build updater
+      run: |
+        make tag
+        make -C scanner bin/updater
+
+    - uses: actions/upload-artifact@v4
+      with:
+        name: updater
+        path: scanner/bin/updater
+        if-no-files-found: error
+
+  run-updater:
+    needs:
+      - build-updater
+    runs-on: ubuntu-latest
+    env:
+      STACKROX_NVD_API_KEY: ${{ secrets.NVD_API_KEY }}
```

Comment:
nit: I know this is copied, but given rate limits we may want to consider separating the key used for dev vs. prod jobs so that PRs with the `pr-update-scanner-vulns` do not count against any rate limits.

Reply:
Makes sense: https://issues.redhat.com/browse/ROX-23802